<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeRuleWidget</class>
 <widget class="QWidget" name="ChargeRuleWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收费规则</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(78, 78, 78，80);
background-color: rgb(247, 235, 183);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
   <item>
    <widget class="QLabel" name="methodLabel">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;仿宋&quot;;</string>
     </property>
     <property name="text">
      <string>收费规则</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox:!hover{
	background-color: rgb(255, 255, 255,30);
	font: 10pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QComboBox:hover{
	background-color: rgb(255, 255, 255,90);
	font: 10pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
       </property>
       <item>
        <property name="text">
         <string>每小时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>年卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>月卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>季卡</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="moneyLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border-width:2px;
	border-style:solid;
	border-color:gray;
	border-radius:10px;
	padding: 2px 2px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;仿宋&quot;;</string>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:!hover{
	font: 10pt &quot;仿宋&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 10pt &quot;仿宋&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 10pt &quot;仿宋&quot;;
	background-color: rgb(0, 0, 0,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
